src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/ExplosionsListener.java

```diff
@@ -41,6 +64,24 @@ public void onEntityExplode(EntityExplodeEvent e) {
 
                     if (blockHandler != null) {
                         success = blockHandler.onBreak(null, block, item, UnregisterReason.EXPLODE);
+                    } else {
+                        item.callItemHandler(BlockBreakHandler.class, handler -> {
+                            if (handler.isExplosionAllowed()) {
+                                BlockStorage.clearBlockInfo(block);
+                                block.setType(Material.AIR);
+
+                                List<ItemStack> drops = new ArrayList<>();
+                                handler.onExplode(block, drops);
+
+                                for (ItemStack drop : drops) {
+                                    if (drop != null && drop.getType() != Material.AIR) {
```

Comment:
should use isAir, if we're doing an air check may as well make it future proof. Who knows what MC, Spigot or other plugins will do